index.js

```diff
@@ -143,6 +144,17 @@ instance.prototype.actions = function (system) {
 				regex: self.REGEX_NUMBER
 			}*/]
 		},
+		'volume_num': {
+			label: 'Volume(specified)',

```

Comment:
Typically this would be something like: 'Set volume'

Reply:
It is tested. Happy to change. Wanted to differentiate from the existing volume up/down function.